notebooks/cytomining-ecosystem/gather-targeted-publication-mentions.ipynb

```diff
@@ -711,17 +706,17 @@
        "</div>"
       ],
       "text/plain": [
-       "  Project Name  Date Created Year  Date Created Year_2  google_scholar_count  \\\n",
-       "0  pycytominer               2019                 2019                    20   \n",
-       "1    CytoSnake               2022                 2022                     0   \n",
-       "2    CytoTable               2022                 2022                     0   \n",
-       "3   IDR_stream               2022                 2022                     0   \n",
+       "  Project Name  Date Created Year  google_scholar_count  bioarxiv_count  \\\n",
+       "0  pycytominer               2019                    20              13   \n",
+       "1    CytoSnake               2022                     0               0   \n",
+       "2    CytoTable               2022                     0               0   \n",
+       "3   IDR_stream               2022                     0               0   \n",
```

Comment:
Let's remove IDR_stream (from this and the other landscape analysis)

Reply:
Got it, thanks - will do. I'll keep the focus isolated for now, but we likely will need to re-run the full workflow as a separate PR.